### PR TITLE
[Kotlin Migration] SimpleCardQueue 

### PR DESCRIPTION
**Pull request by codingtosh**
[Kotlin Migration] SimpleCardQueue

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/libanki/sched/SimpleCardQueue.kt [hunk 1]
```diff
@@ -14,18 +14,12 @@
...
-import androidx.annotation.VisibleForTesting;
-
-class SimpleCardQueue extends CardQueue<Card.Cache> {
-    public SimpleCardQueue(AbstractSched sched) {
-        super(sched);
-    }
-
-    public void add(long id) {
-        add(new Card.Cache(getCol(), id));
+internal class SimpleCardQueue(sched: AbstractSched?) : CardQueue<Card.Cache?>(sched) {
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/libanki/sched/SimpleCardQueue.kt:21]: Cleanup: make sched non-null

**Review by david-allison-1 — CHANGES_REQUESTED** [review]

File: AnkiDroid/src/main/java/com/ichi2/libanki/sched/SimpleCardQueue.kt (hunk 1 above)
Comment (codingtosh) [reply AnkiDroid/src/main/java/com/ichi2/libanki/sched/SimpleCardQueue.kt:23]: Done, PTAL.

**Review by david-allison — APPROVED** [review]
LGTM

**Review by mikehardy — APPROVED** [review]
nice

**Pull request merged**